frontEnd/src/hooks/useRoom.ts

```diff
@@ -61,8 +67,12 @@ const useRoom = (roomId: string) => {
     });
 
     RTCConnection.addEventListener('track', (e) => {
-      setStreamList((prev) => [...prev, { id: socketId, stream: e.streams[0] }]);
+      setStreamList((prev) => {
+        const newArray = [...prev].filter(({ id }) => id !== socketId);
+        return [...newArray, { id: socketId, stream: e.streams[0] }];
+      });
```

Comment:
```suggestion
      setStreamList((prev) => [...prev].filter(({ id }) => id !== socketId).push({ id: socketId, stream: e.streams[0] }));
```
배열 메서드를 활용한다면 이렇게도 표현할 수 있을 것 같아요!

Reply:
push 가 배열에 length에 해당하는 number를 반환해버려서...놀랍게도 number로 set된다고 합니다...!

![image](https://github.com/boostcampwm2023/web05-AlgoITNi/assets/99241871/797ca1be-c9ad-4aae-80c5-6bf38a034dca)

Reply:
Wow 한 수 배워갑니다